doc/manual/modules.rst

```diff
@@ -236,7 +236,29 @@ Namespace miscellanea
 ---------------------
 
 If a name is qualified (e.g. ``Base.sin``), then it can be accessed even if
-it is not exported. This is often useful when debugging.
+it is not exported. This is often useful when debugging. Normally Julia
+variables, functions, types, and the fields of composite types are publicly
+accessible. Julia does not have corresponding "private" versions of these
+items that are inaccessible from outside the module. However, a common
+Julia convention is that names prefixed with an underscore (e.g.,
```

Comment:
Yeah, we were trying it out for a while and there's still a lot of these names hanging around.